packages/style-engine/README.md

```diff
@@ -16,16 +16,48 @@ _This package assumes that your code will run in an **ES2015+** environment. If
 
 This Package is considered experimental at the moment. The idea is to have a package used to generate styles based on a style object that is consistent between: backend, frontend, block style object and theme.json.
 
-Currently it's not a package that generates a wp.styleEngine global because it's not ready yet, it's still a bundled package but ultimately, we want it to be so, once the roadmap is finished:
+Because this package is experimental and still in development it does not yet generate a `wp.styleEngine` global. To get there, the following tasks need to be completed:
 
 **TODO List:**
 
--   Add style definitions for all the currently supported styles in blocks and theme.json.
--   the CSS variable shortcuts for values (for presets...)
--   Support generating styles in the frontend. (Ongoing)
--   Support generating styles in the backend (block supports and theme.json stylesheet). (Ongoing)
--   Refactor all block styles to use the style engine server side. (Ongoing)
--   Refactor all blocks to consistently use the "style" attribute for all customizations (get rid of the preset specific attributes).
+- Add style definitions for all the currently supported styles in blocks and theme.json.
```

Comment:
It looks like the linter is complaining because it wants 3 spaces after the `-` instead of just one 😀

Reply:
_Old man shakes fist at linter!_ 🤣